src/public/interfaces.ts

```diff
@@ -527,12 +527,21 @@ export interface SdkError {
   errorCode: ErrorCode;
   /**
   Optional description for the error. This may contain useful information for web-app developers.
-  This string will not be localized and is not for end-user consumption.
+  This string will not be localized and is not for end-user consumption. 
+  App should not dependend on the string content. The exact value may change. This is only for debugging purposes.
   */
   message?: string;
 }
 
 export const enum ErrorCode {
+  /**
+   * API not supported in the current platform.
+   */
+  NOT_SUPPORTED_ON_PLATFORM = 100,
```

Comment:
There is already an existing error code OLD_PLATFORM = 9000 which checks if platform supports the API or not. This seems to be redundant. What is the difference between this new error code and existing error code 9000?